src/hooks/ChartDataFormatter.ts

```diff
@@ -0,0 +1,61 @@
+/* eslint-disable no-plusplus */
+/* eslint-disable no-array-constructor */
+import { IChartProps } from '../interfaces';
+
+export const chartDataFormatter = ({ dataSet }: IChartProps) => {
+  const label = new Array();
+  const datas = new Array();
+  dataSet.forEach(
```

Comment:
you can use a reduce here  https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/reduce